atomos/atomic.py

```diff
@@ -172,7 +177,7 @@ def __init__(self, value=long(0)):
 
     def __setattr__(self, name, value):
         # Ensure the `_value` attribute is always a long.
-        if name == '_value' and not isinstance(value, types.LongType):
+        if name == '_value' and not isinstance(value, long):
```

Comment:
We can swap `types.LongType` to `long` and keep the other types using `types.*`.